docs/source/index.rst

```diff
@@ -3,8 +3,40 @@
 Composer
 ========
 
+`Composer <https://github.com/mosaicml/composer>`_ is a library for training neural networks better, faster, and cheaper.
+It contains many state-of-the-art methods for accelerating neural network training and improving generalization, along with
+an optional :doc:`Trainer<trainer/using_the_trainer>` API that makes `composing` many different enhancements easy.
+
+We implemented dozens of papers and `trained thousands of models <https://app.mosaicml.com/explorer/imagenet>`_ so you don't have to,
+and built strong baselines to rigorously measure their effect.
+
+With Composer, you can build recipes to:
 .. include:: ../../README.md
    :parser: myst_parser.sphinx_
+   :start-after: Using Composer, you can:
```

Comment:
Is it possible to add new comment lines into the markdown, and use those for the start-after and end-before? Just concerned about changing the text there, and this then unintentionally breaking.